radix-engine-common/src/data/scrypto/custom_well_known_types.rs

```diff
@@ -379,10 +379,15 @@ create_well_known_lookup!(
             MISC_TYPES_START + 7,
```

Comment:
AFAIK, well-known types change is not necessary, so theoretically we could skip it. But since we require the protocol update anyway I would leave it as is.